doc/script_commands.txt

```diff
@@ -1327,7 +1311,6 @@ with other command, such as "if", but often used on its own.
 Label:
 	mes "This will be seen";
 
-Note by FlavioJS: goto's are "evil" and should be avoided if possible (ò_ó)
```

Comment:
Maybe change this to something more general(just an example):
In programming goto has become a no-go and it is common practice not to use it. It should only be used if it really can't be avoided.